contracts/snxToken.ts

```diff
@@ -0,0 +1,14 @@
+export default {
```

Comment:
Can't we just use synthetixJS for that? `ProxyERC20` should be the one to look at

Reply:
+1

Reply:
i dont think we can, because the snxjs instance is locked inside the queries library. I can check again.